lib/decompress/huf_decompress.c

```diff
@@ -218,7 +200,7 @@ size_t HUF_decompress1X2_usingDTable(
 {
     DTableDesc dtd = HUF_getDTableDesc(DTable);
     if (dtd.tableType != 0) return ERROR(GENERIC);
-    return HUF_decompress1X2_usingDTable_internal(dst, dstSize, cSrc, cSrcSize, DTable);
+    return HUF_decompress1X2_usingDTable_internal(dst, dstSize, cSrc, cSrcSize, DTable, /* bmi2 */ 0);
 }
 
 size_t HUF_decompress1X2_DCtx_wksp(HUF_DTable* DCtx, void* dst, size_t dstSize,
```

Comment:
I guess this version is no longer used within `zstd` source code, replaced by the newer `HUF_decompress1X2_DCtx_wksp_bmi2()` ?

Reply:
Yeah, we could delete it. I'm not exactly sure what we want to keep around as part of the Huffman module, since a lot of functions are unused in zstd.

Reply:
That's a good point.
The initial "vision" is that `zstd` would be a client program of `fse` and `hufff0`, and would therefore use the files from these projects unmodified.

This is less and less the case. While `zstd` is actively maintained, `fse` is falling behind, due to lack of attention. Versions start to diverge, we add too many specialized capabilities which are mostly useful for `zstd`, and less for a "generic" entropy interface.

So maybe at some point, it will be better to bite the bullet, and accept that they are now 2 separate code bases, with no coming back.
At this stage, a lot of cleaning will be possible.